recipes/triqs_cthyb/meta.yaml

```diff
@@ -32,14 +32,14 @@ requirements:
     - nfft
     - hdf5
     - openmpi
-    - mpi4py
     - libblas
     - liblapack
     - python
 
 test:
-  imports:
-    - triqs_cthyb 
+  commands:
+    - export OMPI_MCA_btl=self,tcp
+    - OMPI_MCA_plm=isolated OMPI_MCA_rmaps_base_oversubscribe=yes OMPI_MCA_btl_vader_single_copy_mechanism=none mpiexec="mpiexec --allow-run-as-root" python -c "import triqs_cthyb"
```

Comment:
You could move each env variable to a separate line

Reply:
Done